### ISSUE TITLE
Amazon Prime Clone

### ISSUE THREAD

**Issue by Meharsh7804** [you-dont-need/You-Dont-Need-JavaScript#635]
Just Wanted to Add a clone of Amazon Prime Videos Website.

**Comment by tanishkagupta27** [comment]
I want to work on this issue.
Can you please assign it to me.

**Comment by aayush105** [comment]
> Just Wanted to Add a clone of Amazon Prime Videos Website.

OK you can add.

**Comment by Prasoon52** [comment]
hello @aayush105  if this issue is still unsolved can you please assign it to me

**Comment by aayush105** [comment]
> hello [@aayush105](https://github.com/aayush105) if this issue is still unsolved can you please assign it to me

Yes you have already been assigned to Amazon clone project in another issue.

**Comment by Prasoon52** [comment]
@aayush105 sir if this issue is still not solved could you please assign it to me 
i want to work on  Amazon Prime Videos Website.

**Comment by aayush105** [comment]
> [@aayush105](https://github.com/aayush105) sir if this issue is still not solved could you please assign it to me i want to work on Amazon Prime Videos Website.

Sure you can do it.
